devops/ansible-role-girder/tasks/main.yml

```diff
@@ -26,7 +26,8 @@
 - name: Upgrade pip in the virtualenv
   pip:
     name: pip
-    extra_args: "--update"
+    version: ">=21.0.1"
```

Comment:
Is this requirement documented anywhere else?

Reply:
We don't document that we pull in cryptography, and it is a requirement cause by the cryptography module.  How do you recommend we document that an upstream package requires a new version?  We didn't document that we needed pip 9 before this, either (nor check for it).

Reply:
How about here: https://github.com/girder/girder/blob/master/docs/dependencies.rst ?